### PR TITLE
feat: added sorting & pagination to point leaderboard + rpc fix

### PR DESCRIPTION
**Pull request by k0beLeenders**

### CODE REVIEW

File: apps/marginfi-v2-ui/src/components/Points/PointsLeaderBoard/PointsBoardHeader/PointsBoardHeader.consts.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    label: "Lending Points",
+  },
+  {
+    id: "total_activity_borrow_points",
+    numeric: true,
+    disablePadding: false,
+    label: "Borrowing Points",
+  },
+  {
+    id: "total_referral_borrow_points",
```
Comment (edgarpavlovsky) [inline apps/marginfi-v2-ui/src/components/Points/PointsLeaderBoard/PointsBoardHeader/PointsBoardHeader.consts.ts:34]: nit - the `id` here should just generally refer to the head cell (not to what shows in the actual data), but technically this column will be deposit + borrow referral points

**Review by edgarpavlovsky — APPROVED** [review]
@k0beLeenders ltgm overall - left a small comment but should not be blocking

**Review by edgarpavlovsky — CHANGES_REQUESTED** [review]
sorry - few comments after playing around with it live (overall looks good, table is clean!):

* sorting arrow shows up on the right side of words in some cells and left side in others - can we standardize them all to be right side of word?
* let's move to first click on sort button being descending - intuition tells me people are likely to be more interested in who's at the top than the bottom. this will be the opposite for the `rank` column but given rank and total points are the same i think we can just disable rank column sorting
* it'd be nice to show people that they can sort - right now you have to hover over the words to see that the arrow's available. not sure what's aesthetically most pleasing here

other than that, super clean! i love seeing how loading is handled behind the scenes, experience feels quick

**Comment by k0beLeenders** [comment]
Implemented the requested changes @edgarpavlovsky , I can set the arrows to always active if you want?

**Comment by edgarpavlovsky** [comment]
yeah let's set arrows to always active @k0beLeenders

**Review by edgarpavlovsky — APPROVED** [review]

**Comment by edgarpavlovsky** [comment]
image

**Pull request merged**